moped-editor/src/views/projects/projectView/ProjectComponents/utils/useUpdateComponent.js

```diff
@@ -221,7 +221,7 @@ const editReducer = (state, action) => {
 };
 
 export const useUpdateComponent = ({
-  components,
+  projectComponents,
```

Comment:
I renamed `components` to `projectComponents` to differentiate it from `parentComponents`, `siblingComponents`, and `childComponents`.